README.md

```diff
@@ -105,28 +156,40 @@ install.packages(pathTo_tlf.zip, repos = NULL)
 install.packages(pathToOSPSuite.zip, repos = NULL)
 ```
 
-The package also requires the Visual C++ Runtime that is installed with OSPS and can be manually downloaded [here](https://aka.ms/vs/16/release/vc_redist.x64.exe).
-
-## Under Linux
+## On Linux
 
-The **ospsuite** package has been tested under Linux distributions CentOS 7 and Ubuntu 18. Some functionality, such as creating individuals, is not available under Linux. Installation under Linux requires several prerequisites, the detailed instructions can be found in the [Wiki](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/wiki/Setup-ospsuite-R-on-Ubuntu).
-For other Linux distributions Docker containers can be used (Dockerfiles based on CentOS 7 and Ubuntu 18 are available under https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases )
+The **ospsuite** package has been tested under Linux distributions
+**CentOS 7** and **Ubuntu 18**. Some functionality, such as creating
+individuals or populations, is not available under Linux. Installation
+under Linux requires several prerequisites, the detailed instructions
+can be found in the
+[Wiki](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/wiki/Setup-ospsuite-R-on-Ubuntu).
+For other Linux distributions Docker containers can be used (Dockerfiles
+based on CentOS 7 and Ubuntu 18 are available under
+<https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases> )
 
 ## Build from source
 
 You can clone the GIT repository and build the package from source.
 
 ### How to update dependencies from nuget?
 
-- `git submodule update --init --recursive` to install all submodules
-- Make sure you have [ruby](https://www.ruby-lang.org/de/downloads/) install and that it is available in your path
-- Run `rake postclean` or simply double click on `postclean.bat`. This will update all nuget packages and copy the dependencies in the package `inst/lib` folder.
+-   `git submodule update --init --recursive` to install all submodules
+-   Make sure you have [ruby](https://www.ruby-lang.org/de/downloads/)
+    install and that it is available in your path
+-   Run `rake postclean` or simply double click on `postclean.bat`. This
+    will update all nuget packages and copy the dependencies in the
+    package `inst/lib` folder.
 
-# Usage
+# Usage {#usage}
 
-In general, every workflow starts with loading a simulation that has been exported to the `*.pkml` format. The method `loadSimulation()` returns the corresponding simulation that is used as input of other methods. The user can then change values of parameters and initial conditions, run the simulation, and retrieve the simulated results.
+In general, every workflow starts with loading a simulation that has
+been exported to the `*.pkml` format. The method `loadSimulation()`
+returns the corresponding simulation that is used as input of other
+methods. The user can then change values of parameters and initial
+conditions, run the simulation, and retrieve the simulated results.
```

Comment:
do we need this at all?
I would rather just point to the Get Started section of the docu: https://www.open-systems-pharmacology.org/OSPSuite-R/articles/ospsuite.html